scalafmt-sysops/shared/src/main/scala/org/scalafmt/sysops/FileOps.scala

```diff
@@ -80,8 +80,11 @@ object FileOps {
   }
 
   @inline
-  private[sysops] def readAsURL(url: URL)(implicit codec: Codec): String =
-    scala.io.Source.fromURL(url).getLines().mkString("", "\n", "\n")
+  private[sysops] def readAsURL(url: URL)(implicit codec: Codec): String = {
+    val src = scala.io.Source.fromURL(url)
+    try src.getLines().mkString("", "\n", "\n")
+    finally src.close()
```

Comment:
```suggestion
    builder ++= Using.resource(scala.io.Source.fromURL(url))(_.getLines().mkString("", "\n", "\n"))
```
or something along these lines

Reply:
done as well